.github/workflows/create-release-pr.yml

```diff
@@ -1,9 +1,11 @@
 name: Publish new release version
 
+permissions:
+  contents: write
+  pull-requests: write
+
 on:
   push:
-    branches:
-      - "main"
```

Comment:
Is this not required any more?

Reply:
This is a quick wip to try and improve the feedback we get from permissions being insufficient

Reply:
Ah think you might just be doing this for testing